man/oci-image-tool-create.1.md

```diff
@@ -17,8 +17,8 @@ runtime-spec-compatible `dest/config.json`.
 **--help**
   Print usage statement
 
-**--ref**=""
-  The ref pointing to the manifest of the OCI image. This must be present in the "refs" subdirectory of the image. (default "v1.0")
+**--tag**=""
+  The tag pointing to the manifest or the image index of the OCI image. his must be matching 'org.opencontainers.image.ref.name' annotations in index.json. (default "v1.0")
```

Comment:
The spec always uses ref, and I prefer to ref.  And ref must be pointing to a manifest, will not be an image index, we can get the manifest var image index. So I think the change is not correct.